sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -213,6 +226,14 @@ String createStreamIfNeeded() {
         return this.streamName;
       }
 
+      StreamAppendClient generateClient() {
```

Comment:
Minor nit - remove try/catch since the calling function already catches the exception

Reply:
done